src/components-nextjs/Sitemap/SitemapBuilder.ts

```diff
@@ -0,0 +1,56 @@
+import type { SitemapUrl } from './types';
+
+export class SitemapBuilder {
+    private baseUrl: string;
+
+    private urls: SitemapUrl[] = [];
+
+    constructor(baseUrl: string) {
+        this.baseUrl = baseUrl;
+    }
+
+    public addUrl(loc: string) {
```

Comment:
Would be nice to use a full name for this, which is a `location` I assume? This applies to all occurences in this file.

Reply:
Right, will do!